app/Config/api.php

```diff
@@ -20,4 +20,5 @@
     // The number of API requests that can be made per minute by a single user.
     'requests_per_minute' => env('API_REQUESTS_PER_MIN', 180),
 
+    'withoutAuthRoutes' => ['create-shelves','books-report','get-shelves','create-book-and-page'],
```

Comment:
we are making this APIs public?

Reply:
Yes but we have secret key for access this apis key we need to pass secret key in headers